pytket/binders/circuit/Circuit/main.cpp

```diff
@@ -472,15 +472,22 @@ void def_circuit(py::class_<Circuit, std::shared_ptr<Circuit>> &pyCircuit) {
           "\n\n:return: the circuit depth")
       .def(
           "n_gates_of_type",
-          [](const Circuit &circ, const OpType &_type) {
-            return circ.count_gates(_type);
+          [](const Circuit &circ, const OpType &_type, const bool conditional) {
+            if (conditional) {
+              return circ.count_gates(_type) +
+                     circ.count_gates_conditional(_type);
+            } else {
+              return circ.count_gates(_type);
+            }
           },
           "Returns the number of vertices in the dag of a given "
           "operation type.\n\n>>> c.CX(0,1)\n>>> c.H(0)\n>>> "
           "c.CX(0,1)\n>>> c.n_gates_of_type(OpType.CX)\n2\n\n:param "
-          "type: The operation type to search for\n:return: the "
-          "number of operations matching `type`",
-          py::arg("type"))
+          "type: The operation type to search for\n:param "
```

Comment:
```suggestion
          "type: The operation type to search for\n\n:param "
```

Reply:
done in [127a9cc](https://github.com/CQCL/tket/pull/1662/commits/127a9cc5505e32bbeaed7eab2557230f46d05e50)

Reply:
Added this for the return, too.